src/layouts/Header.tsx

```diff
@@ -216,7 +219,7 @@ class Header extends React.Component<IProps, null> {
             </div> : ""
           }
           <div className={css.accountInfo}>
-            { currentAccountAddress ?
+            { currentAccountAddress && currentAccountAddress !== "" ?
```

Comment:
I think this is redundant.  "" is always falsey.

Reply:
This is checking for empty strings, and is not always falsey. Please see this demonstration proving such: 
![image](https://user-images.githubusercontent.com/5522128/82394979-75889780-9a18-11ea-8461-d80f225263e8.png)